src/rebar_prv_as.erl

```diff
@@ -38,7 +38,17 @@ do(State) ->
             {error, "At least one profile must be specified when using `as`"};
         _  ->
             State1 = rebar_state:apply_profiles(State, [list_to_atom(X) || X <- Profiles]),
-            rebar_prv_do:do_tasks(Tasks, State1)
+            {FirstTask, FirstTaskArgs} = hd(Tasks),
```

Comment:
Should we check that Tasks isn't `[]` and have an error message instead of just crashing here if it is? Or is this caught earlier?